docs/cephfs-snap-clone.md

```diff
@@ -0,0 +1,78 @@
+# Create CephFS snapshot and Clone Volume
+
+- [Create CephFS snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
+  - [Prerequisite](#prerequisite)
+  - [Snapshot](#snapshot)
+    - [Create snapshotclass](#create-snapshotclass)
+    - [Create snapshot](#create-snapshot)
+  - [Restore snapshot to a new PVC](#restore-snapshot-to-a-new-pvc)
+  - [Clone PVC](#clone-pvc)
+
+## Prerequisite
+
+- For snapshot functionality to be supported for your Kubernetes cluster, the
+  Kubernetes version running in your cluster should be >= v1.17. We also need the
+  `snapshot controller` deployed in your Kubernetes cluster along with `csi-snapshotter`
+  sidecar container.
+  Refer [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/)
+  for more information on these sidecar controllers. There should
+  be a `volumesnapshotclass` object present in the cluster
+  for snapshot request to be satisified.
+
+## Snapshot
+
+### Create snapshotclass
+
+```console
+$kubectl create -f ../examples/cephfs/snapshotclass.yaml
```

Comment:
add space between $ and kubelet

Reply:
@Madhu-1  as in previous comment https://github.com/ceph/ceph-csi/pull/1343#discussion_r468533502